gulpfile.js/tasks/watch.js

```diff
@@ -12,10 +11,6 @@ var watchTask = function() {
         return PATH_CONFIG.icons
       case 'svgSprite':
         return PATH_CONFIG.icons
-      case 'html':
-        return PATH_CONFIG.html
-      case 'static':
-        return PATH_CONFIG.static
       default:
         return PATH_CONFIG[taskName]
```

Comment:
the removed cases could be run with the default case. they were probably left there from a previous iteration that did not include the default case